src/vs/workbench/contrib/tasks/browser/abstractTaskService.ts

```diff
@@ -2355,7 +2355,7 @@ export abstract class AbstractTaskService extends Disposable implements ITaskSer
 	}
 
 	private async _computeTasksForSingleConfig(workspaceFolder: IWorkspaceFolder, config: TaskConfig.IExternalTaskRunnerConfiguration | undefined, runSource: TaskRunSource, custom: CustomTask[], customized: IStringDictionary<ConfiguringTask>, source: TaskConfig.TaskConfigSource, isRecentTask: boolean = false): Promise<boolean> {
```

Comment:
Not sure when this could happen, but if it is then I would also change the signaure to be `private async _computeTasksForSingleConfig(workspaceFolder: IWorkspaceFolder | undefined)`

Reply:
It looks like it could happen in an empty workspace.